.github/workflows/test_pr.yml

```diff
@@ -34,4 +34,36 @@ jobs:
         with:
           args: deploy -P roster-pr --only hosting:project-dashboard-pull-request
         env:
-          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
\ No newline at end of file
+        FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+
+
+# Only run the coverage once
+      name: get value percentage and print
+      run: |
+       # var REF = 'refs/pull/27/merge.json';
+        REF=${{ github.ref }}
+        # console.log('github.ref: ' + REF);
+        echo "github.ref: $REF"
+        # var PATHS = REF.split('/');
+        IFS='/' read -ra PATHS <<< "$REF"
+        # var CoverageBadge = PATHS[1] + PATHS[2];
+        CoverageBadge="${PATHS[1]}_${PATHS[2]}"
+        # console.log(CoverageBadge); // 'pull_27'
+        echo $CoverageBadge
+        # process.env.CoverageBadge = 'pull_27';
+        echo "CoverageBadge=$(echo ${CoverageBadge})" >> $GITHUB_ENV
+        PERCENTAGE=$(npm test -- --coverage --coverageReporter='text-summary' | grep 'All files' | cut -d '|' -f 5)
+        echo "COVERAGE = $PERCENTAGE" >> $GITHUB_ENV
```

Comment:
Remove spaces around "=". Here you may also have to echo the PERCENTAGE. (I'm not 100% sure about that.)